flink-kubernetes-operator-api/pom.xml

```diff
@@ -55,6 +55,18 @@ under the License.
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-core-api</artifactId>
+            <version>${flink.version}</version>
+            <exclusions>
```

Comment:
Maybe a comment why such exclusion is needed?

Reply:
This hasn't changed compared to what was before here, we already had the exclusions in the api package as we do not require the transitive dependencies for that. The operator core still pulls the transitive deps as well